doc/source/user_guide/ug_case_settings.rst

```diff
@@ -362,12 +364,15 @@ Table of namelist options
    "", "``monthly``", "monthly forcing data", ""
    "", "``ncar``", "NCAR bulk forcing data", ""
    "", "``oned``", "column forcing data", ""
+   "", "``hycom_atm``", "HYCOM atm forcing in NetCDF format", ""
```

Comment:
I would prefer this just be called "hycom" not "hycom_atm". This is how you did it for ocn_data_type and is consistent with the other options (e.g. "ncar" or "clim").